docs/Security.md

```diff
@@ -31,20 +31,33 @@ authentication: {
 ```
 users: [
   {
-    id: "user1"
-    password: "pass"
-    categoryPermissions: {
-      "Category1": ["Read", "Deploy"]
-      "Category2": ["Read", "Write"]
-    }
-    globalPermissions: ["AdminTab"]
+    identity: "user1"
```

Comment:
I think that we need to start updating `BreakingChanges.md`...

Reply:
In future.. :)